applications/osb-portal/src/pages/RepositoryPage.tsx

```diff
@@ -293,7 +293,7 @@ export const RepositoryPage = (props: any) => {
                 </Box>
               </Grid>
               <Grid item={true} xs={12} md={6} className="verticalFill">
-                <Box className="verticalFit">
+                <Box className="verticalFit" style={{ width: '100%' }}>
```

Comment:
why using inline style here?

Reply:
I thought that since it is a minor change, an inline style would keep the page leaner than adding a new CSS rule.